Test/metatests/StdLibsOffByDefaultInTests.dfy

```diff
@@ -0,0 +1,17 @@
+
+// RUN: %exits-with 2 %verify "%s" > "%t"
```

Comment:
Can you add a test uses the standard libraries?

Reply:
This one does in fact test that it's possible when `--allow-standard-libraries` is provided, on the next LIT command line. But that's definitely not very clear, I will add a separate direct test.